lib/modules/dosomething/dosomething_api/dosomething_api.module

```diff
@@ -4,12 +4,34 @@
  * Code for the dosomething_api feature.
  */
 
-include_once 'dosomething_api.features.inc';
-include_once 'resources/campaign_resource.inc';
-include_once 'resources/member_resource.inc';
-include_once 'resources/signup_resource.inc';
-include_once 'resources/reportback_resource.inc';
-include_once 'resources/term_resource.inc';
+module_load_include('inc', 'dosomething_api', 'dosomething_api.features');
+module_load_include('inc', 'dosomething_api', 'resources/campaign_resource');
+module_load_include('inc', 'dosomething_api', 'resources/member_resource');
+module_load_include('inc', 'dosomething_api', 'resources/reportback_resource');
+module_load_include('inc', 'dosomething_api', 'resources/reportback_item_resource');
+module_load_include('inc', 'dosomething_api', 'resources/reportback_file_resource');
+module_load_include('inc', 'dosomething_api', 'resources/signup_resource');
+module_load_include('inc', 'dosomething_api', 'resources/term_resource');
```

Comment:
I would vote to leave these at `include_once` since we are just including files. it's the [standard we've been using to include files](https://github.com/DoSomething/dosomething/search?utf8=%E2%9C%93&q=include_once)